### PR TITLE
Require newline before return

### PR DESCRIPTION
**Pull request by ostrolucky**

### CODE REVIEW

**Comment by Majkl578** [comment]
IMO this case should be allowed:
```php
public function foo(bool $foo) : int
    if ($foo) {
        return 123;
    }

    return 456;
}
```

**Comment by ostrolucky** [comment]
@Majkl578 that is allowed, otherwise there would be more violations

File: lib/Doctrine/ruleset.xml [hunk 1]
```diff
@@ -202,6 +202,14 @@
     <rule ref="SlevomatCodingStandard.Commenting.RequireOneLinePropertyDocComment"/>
     <!-- Forbid assignments in conditions -->
     <rule ref="SlevomatCodingStandard.ControlStructures.AssignmentInCondition"/>
+    <!-- Require consistent spacing for control structures -->
+    <rule ref="SlevomatCodingStandard.ControlStructures.ControlStructureSpacing">
```
Comment (carusogabriel) [inline lib/Doctrine/ruleset.xml]: Are the @doctrine/coding-standard-approvers team okay of approving one by one of this Sniff in particular? 'cause I'd like to propose some new lines after some tokens in specific, as well :)

Comment (Ocramius) [reply lib/Doctrine/ruleset.xml]: You can send a cumulative patch, it just lowers the chance of merge due to more possible discussions

Comment (ostrolucky) [reply lib/Doctrine/ruleset.xml]: This. Originally I wanted to introduce this with default settings, but it requires spaces around yields, which night be controversial and I don't like it personally either

**Pull request merged**